### PR TITLE
Footers refactoring

### PR DESCRIPTION
**Pull request by KateM-qa**
1. Invoked autoFillOutForm method for sumFooterTest, minFootersTest, averageFooterTest and saveFooterTest

2. Implemented methods getSumValueString, getMinValueString, getAverageValueString which do calculations based on current random generated values (not hardcoded) and compare them with the form values to make sure they match.

3. Organized FootersEditPage - removed unused methods/variables

4. There are still a few tests written by other people and not using methods described above. I will ask them if they want to refactor after/if my changes will be accepted by reviewers and merged.

### CODE REVIEW

File: src/test/resources/local.properties.TEMPLATE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 serverCredentials=false
 
-default.username=user1@tester.com
-default.password=ah1QNmgkEO
+default.username=user180@tester.com
+default.password=UNWDnZkJ9O
```
Comment (oooleks) [inline src/test/resources/local.properties.TEMPLATE:4]: You are not suppose to edit this template file - please revert

File: src/test/resources/local.properties.TEMPLATE [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-marketplace.username=tester1@platformatica.net
-marketplace.password=d8aBFiEH31
\ No newline at end of file
+marketplace.username=tester180@platformatica.net
+marketplace.password=VwmAAQWCEf
```
Comment (oooleks) [inline src/test/resources/local.properties.TEMPLATE:7]: You are not suppose to edit this template file - please revert

**Review by oooleks — CHANGES_REQUESTED** [review]

File: src/test/resources/local.properties.TEMPLATE (hunk 1 above)
Comment (KateM-qa) [reply src/test/resources/local.properties.TEMPLATE]: Sorry, reverted

File: src/test/resources/local.properties.TEMPLATE (hunk 2 above)
Comment (KateM-qa) [reply src/test/resources/local.properties.TEMPLATE]: Reverted

**Review by oooleks — APPROVED** [review]

**Review by guram21 — APPROVED** [review]

**Review by Burdishka — APPROVED** [review]

**Pull request merged**